lib/registry.js

```diff
@@ -27,7 +28,7 @@ var getDoc = function(change, callback) {
         // if there's no json and rev # is 1, then it's a brand new package, so
         // just wait and try again. ignore if deleted
         var retry;
-        if (change.deleted) {
+        if (retries >= 5 || change.deleted) {
```

Comment:
We should emit a log entry "retry limit reached for {pkg} revision {rev}" or something like that.